taskslist/app/views/tasks/index.html.erb

```diff
@@ -1,12 +1,38 @@
 <h1>Here are the tasks</h1>
 
-<table class = "table table-hover">
+<table class="table table-hover">
   <tr>
     <th>
       Task
     </th>
+    <th>Completed?</th>
+    <th>Delete</th>
+    <th>Edit</th>
   </tr>
   <% @tasks.each do |task| %>
-    <tr><td><%= link_to "#{task.name}", "/tasks/#{task.id}" %></td></tr>
+    <tr>
+      <td>
+        <%= link_to "#{task.name}", "/#{task.id}" %>
+      </td>
+      <% if task.done_date == nil %>
```

Comment:
In general, it is better to use the `.nil?` method rather than checking the `nil` directly